testcafe/tests/pages/photos-model.js

```diff
@@ -61,20 +68,21 @@ export default class Page {
 
   async waitForLoading() {
     await t.expect(this.loading.exists).notOk('Page still loading')
+    await isExistingAndVisibile(this.contentWrapper, 'Content Wrapper')
   }
 
   //@param {string} when : text for console.log
   async getPhotosCount(when) {
     let allPhotosCount
-
-    if (await this.photoEmpty.exists) {
+    if ((await this.folderEmpty.exists) && (await this.folderEmpty.visible)) {
+      await isExistingAndVisibile(this.folderEmpty, 'Photo Empty section')
```

Comment:
`isExistingAndVisibile` already checks if `exists` and `visible` no ? Why doing it twice? 

if(await isExistingAndVisibile(this.folderEmpty, 'Photo Empty section')){
allPhotoCount = 0
} 

?